### ISSUE TITLE
Enhancement: UserProfile import of additional Azure AD attributes

### ISSUE THREAD

**Issue by rjanssen2** [SharePoint/PnP#1758]
#### Category
[ ] Bug
[x] Enhancement

#### Environment
[x] Office 365 / SharePoint Online
[ ] SharePoint 2016
[ ] SharePoint 2013

#### Expected or Desired Behavior
I am looking for a UserProfile bulk update solution that can run in parallel to the existing limited
Azure AD to SPO directory sync, to replicate other attributes in addition to the fixed set of
attributes already replicated.  Hopefully this would work without explicitly creating and uploading
a file and importing it via the API.   Of course this solution could use that method internally.
All the attributes I want to import already are in Azure AD, synchronized from on-premise AD.

**Comment by spdevdocs** [comment]
We closed this issue as it had no activity within the last 180 days. This is a generic process we have decided to perform for issues, which have not been explicitly marked still to be "work in progress" based on tags. We are performing this cleaning to make sure that old issues that have already been solved (but not closed) or are no longer relevant are cleaned out and make the issues more manageable. If this issue still valid, we would ask you to **open a new issue** and follow the guidance in the issue template related on the recommended location. We do apologize for any inconveniences this might cause. Please do remember that issues in the issue lists are also messages for others in the community, so you can also check if you can assist on any of them. *“Sharing is caring!”*
